### ISSUE TITLE
bug: Forecastle crd spec.urlFrom.ingressRef.name does not work

### ISSUE THREAD

**Issue by servo1x** [stakater/Forecastle#171]
We're using Forecastle v1.0.68, everytime we try to apply the forecastle crd with `spec.urlFrom.ingressRef.name: xxx`, it gets ignored.

Attempt:

```yaml
apiVersion: forecastle.stakater.com/v1alpha1
kind: ForecastleApp
metadata:
  name: "test-1"
  namespace: test
spec:
  name: "test-app"
  group: "test-group"
  icon: ""
  networkRestricted: false
  instance: "test1"
  urlFrom:
    ingressRef:
      name: "test-ingress"
```

After it gets applied:

```yaml
apiVersion: forecastle.stakater.com/v1alpha1
kind: ForecastleApp
metadata:
  name: "test-1"
  namespace: test
spec:
  name: "test-app"
  group: "test-group"
  icon: ""
  networkRestricted: false
  instance: "test1"
  urlFrom:
    ingressRef:
      name: {}
```

**Comment by Hounddog** [comment]
I have added a pull Request for this issue #176

**Comment by davidkarlsen** [comment]
Fixed in #184.
